templates/web/bathnes/admin/bodies/_category_field.html

```diff
@@ -0,0 +1,17 @@
+[% IF contact.in_storage %]
+    <h1>[% contact.category | html %]</h1>
+    <input type="hidden" name="category" value="[% contact.category | html %]" >
+
+    <p>
```

Comment:
An `admin-hint` here would be nice, just to explain what happens when a display name is set/changed.